integration_tests/src/main/python/ast_test.py

```diff
@@ -381,8 +381,9 @@ def test_or(data_gen):
                        f.lit(False) | f.col('b'),
                        f.col('a') | f.col('b')))
 
+@ignore_order
 def test_multi_tier_ast():
     assert_gpu_ast(
         is_supported=True,
         func=lambda spark: spark.range(10).withColumn("x", f.col("id")).repartition(1)\
```

Comment:
Why do we need the rapartition? Removing it would produce the same result because project preserves order.

Reply:
It's there to avoid Spark from simplifying the expression.  I added a comment to the test.